frontend/views/containers/group-settings/GroupRulesSettings.vue

```diff
@@ -1,42 +1,35 @@
 <template lang='pug'>
-  ul.c-wrapper(data-test='votingRules')
-    li.cardBox.c-card(
-      v-for='rule in votingRulesSorted'
-      :class='{ isActive: isRuleActive(rule) }'
-      :aria-current='isRuleActive(rule)'
-      :data-test='rule'
-    )
-      p.is-title-4 {{ config[rule].title }}
-      p.has-text-1.c-expl(v-safe-html='config[rule].explanation')
-      i18n.pill.is-primary.c-active(v-if='isRuleActive(rule)') Active
-
-      dl.c-status(v-if='isRuleActive(rule)')
-        dt.c-status-term {{ config[rule].status }}
-        dd.c-status-desc
-          span(v-safe-html='votingValue(rule)' data-test='ruleStatus')
-          button.link(
-            tag='button'
-            data-test='changeRule'
-            @click='openVotingProposal(rule)'
-          ) {{ groupShouldPropose ? L('Propose change') : L('Change') }}
-
-      banner-simple.c-banner(
-        severity='info'
-        data-test='ruleAdjusted'
-        v-if='groupShouldPropose && votingRuleAdjusted(rule)'
-      ) {{ votingRuleAdjusted(rule) }}
-
-      button.link(
-        v-if='!isRuleActive(rule)'
-        data-test='changeRule'
-        @click='openVotingProposal(rule)'
-      ) {{ groupShouldPropose ? L('Propose changing to this system') : L('Change to this system') }}
+ul.c-wrapper(data-test='votingRules')
+  li.cardBox.c-card.is-active(
+    :data-test='config.rule'
+    :aria-current='true'
+  )
+    p.is-title-4 {{ config.title }}
+    p.has-text-1.c-expl(v-safe-html='config.explanation')
+    i18n.pill.is-primary.c-active Active
+
+    dl.c-status
+      dt.c-status-term {{ config.status }}
+      dd.c-status-desc
+        span(v-safe-html='votingValue(config.rule)' data-test='ruleStatus')
+
+    button.link(
+      type='button'
+      data-test='changeRule'
+      @click='openVotingProposal'
+    ) {{ groupShouldPropose ? L('Propose change') : L('Change') }}
+
+  banner-simple.c-banner(
+    serverity='info'
+    data-test='ruleAdjusted'
+    v-if='groupShouldPropose && votingRuleAdjusted()'
+  ) {{ votingRuleAdjusted() }}
```

Comment:
@SebinSong I'm sorry I guess I wasn't clear enough. We don't want to *rewrite* the code so that this isn't here anymore, because we might want to add this feature back in the future. We just want to comment out the old code that shows that section, or disable it with a `v-if='false'`.

That way, whenever we do fix the issues with the disagreement rule, it is simple to re-enable it.